### ISSUE TITLE
[New Rule] Replace f-string without any text with using value directly instead

### ISSUE THREAD

**Issue by simon-liebehenschel** [MartinThoma/flake8-simplify#71]
## Explanation

Analyze f-strings. If f-string has no text and only a passed value, then use that value directly.

# Example 1

```python
# Bad

foo = "bar"
my_text = f"{foo}"
```
```python
# Good

foo = "bar"
my_text = foo
```
# Example 2

```python
# Bad

def my_func(foo: str):
    my_text = f"{foo}"
```
```python
# Good
def my_func(foo: str):
    my_text = foo
```
# Example 3

```python
# Bad

my_number = 123
my_text = f"{foo}"
```
```python
# Good

my_number = 123
my_text = str(foo)
```

**Comment by rpdelaney** [comment]
I just came here to request this exact feature!

> Could you please let me know whether your plugin can detect types or I need to address this feature implementation to Mypy?

Maybe I misunderstand. Why would simplify need to perform type checking to know that this pattern is incorrect?

**Comment by AIGeneratedUsername** [comment]
> Maybe I misunderstand. Why would simplify need to perform type checking to know that this pattern is incorrect?

You are right. Probably no type checking is required.

**Comment by MartinThoma** [comment]
```
astpretty --no-show-offsets /dev/stdin <<< `cat example1.py`
Module(
    body=[
        Assign(
            targets=[Name(id='foo', ctx=Store())],
            value=Constant(value='bar', kind=None),
            type_comment=None,
        ),
        Assign(
            targets=[Name(id='my_text', ctx=Store())],
            value=JoinedStr(
                values=[
                    FormattedValue(
                        value=Name(id='foo', ctx=Load()),
                        conversion=-1,
                        format_spec=None,
                    ),
                ],
            ),
            type_comment=None,
        ),
    ],
    type_ignores=[],
)
```

**Comment by MartinThoma** [comment]
```
$ astpretty --no-show-offsets /dev/stdin <<< `cat bad-example3.py`
Module(
    body=[
        Assign(
            targets=[Name(id='my_number', ctx=Store())],
            value=Constant(value=123, kind=None),
            type_comment=None,
        ),
        Assign(
            targets=[Name(id='my_text', ctx=Store())],
            value=JoinedStr(
                values=[
                    FormattedValue(
                        value=Name(id='foo', ctx=Load()),
                        conversion=-1,
                        format_spec=None,
                    ),
                ],
            ),
            type_comment=None,
        ),
    ],
    type_ignores=[],
)
```
